src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -138,26 +135,21 @@ public virtual Task InitializeAsync()
 
                     if (doCommit) transaction.Commit();
                 }
-
-                this.logger.LogTrace("(-)");
             });
 
-            this.logger.LogTrace("(-)");
             return task;
         }
 
         /// <inheritdoc />
         public Task<Transaction> GetTrxAsync(uint256 trxid)
         {
-            this.logger.LogTrace("({0}:'{1}')", nameof(trxid), trxid);
```

Comment:
@noescape00 how does this handle these situations where we log entry parameters?

Reply:
Fody would insert same log as the one that is deleted on the line you've commented on. 
It will be inserted on compilation. 
For more info check #2249